RevitLookup/Snoop/ModelessWindowFactory.cs

```diff
@@ -38,7 +38,25 @@ public static void Show(Form newForm, Document targetDocument = null, Form paren
 
             if (targetDocument != null && newForm is IHaveCollector formWithCollector) formWithCollector.SetDocument(targetDocument);
             newForm.Show(new ModelessWindowHandle(parentForm));
-            newForm.FormClosed += (s, e) => ModelessWindowHandle.BringRevitToFront();
+            newForm.FormClosed += (s, e) =>
+            {
+                ModelessWindowHandle.BringRevitToFront();
+                FocusOwner(s as Form);
```

Comment:
I think you need to use direct cast here, because calling `form.Owner` in the FocusOwner method, in case of null will call NPE) or write ` form?.Owner`